code/game/objects/items/crayons.dm

```diff
@@ -483,7 +483,7 @@
 
 	var/temp = "rune"
 	var/ascii = (length(drawing) == 1)
-	if(ascii && is_alpha(drawing))
+	if(ascii && is_lowercase_character(drawing))
```

Comment:
Won't this filter out people typing in uppercase?

Reply:
This code is unchanged, I just renamed the macro